tests/phpunit/ApiTest.php

```diff
@@ -99,4 +138,29 @@ private function httpClientWillReturnResponse(MockObject $request)
         $response = $this->getMockBuilder(ResponseInterface::class)->getMock();
         $this->httpClient->expects($this->once())->method('sendRequest')->with($request)->willReturn($response);
     }
+
+    private function getFakeConfig(): Config
+    {
+        $configVals = [
+            'hello-i-am-a-token',
+            null,
+            false,
+            false,
+            'api.upmind.io',
+            'https',
+            false
+        ];
+
+        return new class(...$configVals) extends Config {
+            private ?string $overrideBrandId = null;
```

Comment:
We already have `$brandId` supported here which can be passed to the Config constructor without these extra steps

Reply:
I'm afraid that wouldn't work in this case, as Config is an immutable object. We can contrive our own mock, or use a PHPUnit mock object, but we cannot use a real object and override the setting after construction.